aws/database/main.tf

```diff
@@ -81,7 +81,8 @@ resource "aws_db_instance" "default" {
   publicly_accessible    = var.publicly_accessible
   multi_az               = var.multi_az
   ca_cert_identifier     = var.ca_cert_identifier
-  apply_immediately      = var.apply_immediately
+  apply_immediately      = var.apply_immediately 
+	parameter_group_name   = var.parameter_group_name != "" ? var.parameter_group_name : "default.${var.type}${local.ver}"
```

Comment:
Can you run this through `terraform fmt`

Reply:
Done, thanks for catching this and sorry for miss it